.github/workflows/cache.yml

```diff
@@ -0,0 +1,26 @@
+name: Cache
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/cache@v1.1.2
+      id: cache
+      if: startsWith(runner.os, 'Linux')
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/pip-dep/requirements*.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install Dependencies
+      if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Once the caching step is added to the other yml files, their installation steps will need this check.